src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/CompilerServices/AnalysisCharacteristicAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.CompilerServices
+{
+    // When applied to an intrinsic method, the method will become a characteristic check.
```

Comment:
Can we expand on this comment (say something like "the method name is the name of the characteristic calculated in the ILCompiler, see `AnalysisCharacteristicNode` for more information")?